rest-api/dao/participant_dao.py

```diff
@@ -274,10 +274,10 @@ def add_missing_hpo_from_site(self, session, participant_id, site_id):
     participant = self.get_for_update(session, participant_id)
     if participant is None:
       raise BadRequest('No participant %r for HPO ID udpate.' % participant_id)
-    if participant.hpoId == site.hpoId:
```

Comment:
We should probably replace this with
if participant.siteId == site.siteId:
  return

(The idea being we don't have to mess with anything on the participant or summary if the site hasn't changed.)

Reply:
done